### DISCUSSION TITLE
Deleting a repository

### DISCUSSION THREAD

**Discussion by Keith-43** [community/community#23392] [New to GitHub]
<p>I am experimenting in how to use GitHub.  I created a personal repository and fooled around with it.  Now I am ready to get serious.  How do I delete the “practice” repository that I have played around with?  With my repository open, I don’t see a “Delete” button.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Go to the repository settings, and scroll down to the section marked as “Danger Zone”. There should be a delete button in there.</p>

**Comment by ahnaf889** [comment]
Delete reposatray
